doc/manual.org

```diff
@@ -513,6 +512,31 @@ dependencies. See the [[User actions][actions section]] for more details.
    (action  (chdir ${ROOT} (run ${bin:ocamlyacc} ${<})))))
 #+end_src
 
+**** menhir
+
+The basic form for defining menhir parsers (analogous to ocamlyacc):
+
+#+begin_src scheme
+(menhir
+ ((modules (<parser1> <parser2> ...))))
+#+end_src
+
+Modular parsers (--base flag) can be defined by adding the `merge_into` option:
```

Comment:
This is using markdown syntax, in org-mode it should be `=--base=` and `=merge_into=`. I think that the doc should also make it clear that in the first case several OCaml modules are generated while in the second only one is.